Projekty/Projekt1/VinichenkoYevheniiWolnyKrzysztof/feature_engineering.ipynb

```diff
@@ -0,0 +1,1463 @@
+{
```

Comment:
<p>Fajnie jakbyście napisali więcej o Cramerze</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Może warto powyższą heatmapę zrobić Cramerem, bo Pearson to tak średnią dla zmiennej binarnej</p>